modules/csg/csg_shape.cpp

```diff
@@ -929,7 +929,12 @@ CSGBrush *CSGMesh3D::_build_brush() {
 
 void CSGMesh3D::_mesh_changed() {
 	_make_dirty();
-	update_gizmos();
+
+	if (likely(is_accessible_from_caller_thread())) {
+		update_gizmos();
+	} else {
+		call_deferred(SNAME("update_gizmos"));
```

Comment:
```suggestion
		callable_mp((Node3D *)this, &Node3D::update_gizmos).call_deferred();
```
Use callable syntax if possible

Reply:
Thanks! And Merry Christmas ~

Reply:
You too!